services/web/server/src/simcore_service_webserver/socketio/_handlers.py

```diff
@@ -109,9 +114,8 @@ async def connect(
     _logger.debug("client connecting in room %s", f"{socket_id=}")
 
     try:
-        user_id = await _authenticate_user(
-            socket_id, app, request=environ["aiohttp.request"]
-        )
+        auth_user_handler = auth_user_factory(socket_id)
+        user_id = await auth_user_handler(environ["aiohttp.request"])
```

Comment:
Q: why was this changed? It is not clear to me

Reply:
handlers that are decorated with `@login_required` must have a signature like `handler( request: web.Request)`. The sockets did not have that and I converted to comply